docs/dataclients/kubernetes.md

```diff
@@ -240,6 +345,11 @@ and **my-app-2** will get **20%** of the traffic:
               servicePort: http
             path: /
 
+# Filters
+
+Filters can modify http requests and responses. There are a plenty of
```

Comment:
Potentially better but not sure: "There are plenty of things..." (without the "a")